RELEASE.md

```diff
@@ -22,68 +22,72 @@ This page describes the release process and the currently planned schedule for u
-For a release candidate, append something like `-rc.0` to the version (with the corresponding changes to the tag name, the release name etc.).
+## Prepare your release
 
-For release candidates still update `CHANGELOG.md`, but when you cut the final release later, merge all the changes from the pre-releases into the one final update.
+For a new major or minor release, work from the `main` branch. For a patch release, work in the branch of the minor release you want to patch (e.g. `release-0.1` if you're releasing `v0.1.1`).
 
-Entries in the `CHANGELOG.md` are meant to be in this order:
+Add an entry for the new version to the `CHANGELOG.md` file. Entries in the `CHANGELOG.md` should be in this order:
 
 * `[CHANGE]`
 * `[FEATURE]`
 * `[ENHANCEMENT]`
 * `[BUGFIX]`
 
-### 2. Draft the new release
+Create a PR for the changes to be reviewed.
 
-Tag the new release via the following commands:
+## Publish the new release
 
+For new minor and major releases, create the `release-<major>.<minor>` branch starting at the PR merge commit.
+From now on, all work happens on the `release-<major>.<minor>` branch.
+
+Bump the version in the `VERSION` file in the root of the repository.
+
+Regenerate setup.yaml based on latest code and then commit the changed bundle.yaml to the `release-<major>.<minor>` branch:
 ```bash
-$ tag="v$(< VERSION)"
-$ git tag -s "${tag}" -m "${tag}"
-$ git push origin "${tag}"
+make manifests
+git add ./
+git commit -s -m "regenerate setup.yaml"
+git push
 ```
 
-Optionally, you can use this handy `.gitconfig` alias.
+Manually build and push the container image:
 
-```ini
-[alias]
-  tag-release = "!f() { tag=v${1:-$(cat VERSION)} ; git tag -s ${tag} -m ${tag} && git push origin ${tag}; }; f"
+```bash
+make build-op-amd64 -e FO_IMG=<image of fluent operator>
+docker push <image of fluent operator>
+make build-fb-amd64 -e FB_IMG=<image of fluent bit>
+docker push <image of fluent bit>
+make build-fd-amd64 -e FD_IMG=<image of fluentd>
+docker push <image of fluentd>
 ```
 
-Then release with `git tag-release`.
+Currently, a CI pipeline automatically pushed the container images to [docker hub](https://hub.docker.com/repository/docker/kubesphere).
 
-### 3. Build and push image
+Tag the new release with a tag named `v<major>.<minor>.<patch>`, e.g. `v2.1.3`. Note the `v` prefix. You can do the tagging on the commandline:
 
 ```bash
-make build
-```
\ No newline at end of file
+tag="$(< VERSION)"
+git tag -a "${tag}" -m "${tag}"
+git push origin "${tag}"
+```
+Commit all the changes.
+
+Go to https://github.com/fluent/fluent-operator/releases/new, associate the new release with the before pushed tag, paste in changes made to `CHANGELOG.md`, add file `setup.yaml` from `manifests/setup/setup.yaml` and then click "Publish release".
+
+For patch releases, submit a pull request to merge back the release branch into the `master` branch.
```

Comment:
=>
For patch releases, cherry-pick the commits from the release branch into the `master` branch.